src/hera/value_from.py

```diff
@@ -46,6 +48,7 @@ class ValueFrom:
     json_path: Optional[str] = None
     parameter: Optional[str] = None
     path: Optional[str] = None
+    supplied: Optional[bool] = None
```

Comment:
Should this be a bool only instead? It seems like it doesn't matter if the value is set to True or False or not.